music/src/ddsp/audio_utils.ts

```diff
@@ -0,0 +1,117 @@
+/**
```

Comment:
Yah, there's nothing ddsp specific about these. It might be nice to centralize all the audio_utils in the repo if that makes sense.